File: tests/fixtures/sample_java/Widget.java
package com.example;

import java.util.List;
import java.util.ArrayList;

/** A small widget that counts things. */
public class Widget {
    private int count;

    /** Increments the counter by n and returns the result. */
    public int bump(int n) {
        return helper(n);
    }

    static int helper(int x) {
        return x + 1;
    }
}

File: tests/fixtures/sample_rust/widget.rs
use std::fmt;

/// A widget struct that counts things.
pub struct Widget {
    count: i32,
}

impl Widget {
    /// Bumps the count by n and returns it.
    pub fn bump(&self, n: i32) -> i32 {
        helper(n)
    }
}

/// Greets the given name.
pub fn greet(name: &str) -> String {
    String::from(name)
}

trait Speak {
    fn speak(&self);
}
